pkg/scheduler/framework/util.go

```diff
@@ -262,3 +262,16 @@ func (nl *NodeLister) List() ([]*v1.Node, error) {
 	}
 	return nodes, nil
 }
+
+func isNodeUnschedulable(node *v1.Node) bool {
```

Comment:
node could be nil, so better to add a nil check here. Refer to https://github.com/volcano-sh/volcano/blob/master/pkg/scheduler/api/node_info.go#L164C3-L164C18

Reply:
done